### ISSUE TITLE
Feature request: checkbox for raft

### ISSUE THREAD

**Issue by ccfoo242** [prusa3d/PrusaControl#76]
I would like to be able to use rafts and not just brim.

**Comment by prusa3d-bb** [comment]
Hello Steve,

The raft option is on Slic3r Prusa Edition too.



Regards,

—
Blas Palencia
Customer Support

PRUSA Research
+420 222 263 718
+421 220 570 305
188/7a Partyzánská, 17000, Prague
shop.prusa3D.com

Please rate our reply ...be generous :-)

--- original message ---
On Mon, Feb 12, 2018 at 09:39 pm, <notifications@github.com> Steve Bloomfield wrote:


I would like to be able to use rafts and not just brim.

—

You are receiving this because you are subscribed to this thread.

Reply to this email directly, view it on GitHub, or mute the thread.
--- end of original message ---

**Comment by ccfoo242** [comment]
Yes, I am able to use that, but PrusaControl has an easier/nicer interface so it would be nice to have a raft option next to the brim option.

**Comment by prusa3d-bb** [comment]
Hello,

We are working in Prusa Control and will be improved with new features in the future.

Have a nice weekend!


—
Blas Palencia
Customer Support

PRUSA Research
+420 222 263 718
+421 220 570 305
188/7a Partyzánská, 17000, Prague
shop.prusa3D.com

Please rate our reply ...be generous :-)

--- original message ---
On Wed, Feb 14, 2018 at 06:11 pm, <notifications@github.com> Steve Bloomfield wrote:


Yes, I am able to use that, but PrusaControl has an easier/nicer interface so it would be nice to have a raft option next to the brim option.

—

You are receiving this because you commented.

Reply to this email directly, view it on GitHub, or mute the thread.
--- end of original message ---

**Comment by vitriolix** [comment]
👍

**Comment by cwei** [comment]
👍

Viele Grüße
Carsten

**Comment by crua9** [comment]
This is a needed feature
